ajax/libs/octicons/package.json

```diff
@@ -7,7 +7,7 @@
     "OFL-1.1",
```

Comment:
Please help update license, only `MIT` is available now, this change can be with `homepage` update.
Ref: https://github.com/primer/octicons/blob/master/LICENSE